### PR TITLE
Fix: navigation buttons shouldn't cover scrollbar

### PR DESCRIPTION
**Pull request by ConradJChan**
Made the navigation buttons skinner, in particular the next file navigation button, in order to allow room for the preview content scrollbar to be clickable

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/lib/_navigation.scss [hunk 1]
```diff
@@ -39,6 +39,11 @@ $navigationBtnWidth: 50px;
         box-shadow: none;
         outline: none;
     }
+
+    & > svg {
+        margin-left: -5px;
```
[the remark this answers is not in the document]
Comment (ConradJChan) [reply src/lib/_navigation.scss]: This is to center the SVG within the now 40px button, stand by for screenshots

File: src/lib/_navigation.scss [hunk 2]
```diff
@@ -52,9 +57,10 @@ $navigationBtnWidth: 50px;
 .bp-navigate-left {
     border-radius: 0 2px 2px 0;
     left: 0;
```
Comment (jeremypress) [inline src/lib/_navigation.scss]: remove this?

File: src/lib/_navigation.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-$navigationBtnWidth: 50px;
+$navigationBtnWidth: 40px;
```
Comment (DanDeMicco) [inline src/lib/_navigation.scss]: Why reduce the button width? What if you left everything the same, but changed the left and right of .bp-navigate-left and .bp-navigate-right, would that work?

**Comment by ConradJChan** [comment]
Before:
before

After:
after

File: src/lib/_navigation.scss (hunk 3 above)
Comment (ConradJChan) [reply src/lib/_navigation.scss]: Yeah, you're right I could keep the button width the same, I guess I was just trying to keep the button about in the same place as it ever was (though what is 5px vs 10px really)

**Comment by ConradJChan** [comment]
In addressing @DanDeMicco's comment, I'm keeping the button the same size now. I've also increased the offset to 20px because across the different browsers the scroll bar width is 17px, so just rounding up to 20 to be safe

Now looks like this:
after 20px offset

Thoughts?

File: src/lib/_navigation.scss [hunk 4]
```diff
@@ -39,6 +39,10 @@ $navigationBtnWidth: 50px;
         box-shadow: none;
         outline: none;
     }
+
+    & > svg {
```
Comment (jstoffan) [inline src/lib/_navigation.scss]: Is this still necessary?

Comment (ConradJChan) [reply src/lib/_navigation.scss]: nope it is not, will remove

**Pull request merged**